model/src/main/java/org/perfrepo/dto/test_execution/TestExecutionGroupValueDto.java

```diff
@@ -0,0 +1,29 @@
+package org.perfrepo.dto.test_execution;
+
+import java.util.List;
+
+/**
+ * @author Jiri Grunwald (grunwjir@gmail.com)
+ */
+public class TestExecutionGroupValueDto {
```

Comment:
Could you please explain purpose of this class? I would also rename it to ValueGroupDto, you don't have to put "TestExecution" everywhere